lib/python/rose/macros/__init__.py

```diff
@@ -22,12 +22,12 @@
 """
 
 import rose.macro
-import compulsory
-import duplicate
-import format
-import rule
-import trigger
-import value
+from . import compulsory
```

Comment:
Curious, can Python3 not handle these imports any more?

Reply:
I believe it's that 2to3 doesn't like them - this is the preferred style.

Reply:
This way is probably safer, less risk of namespace clashes.